crates/trie/sparse-parallel/src/trie.rs

```diff
@@ -587,6 +592,9 @@ impl SparseTrieInterface for ParallelSparseTrie {
         let mut prefix_set = core::mem::take(&mut self.prefix_set).freeze();
         let (subtries, unchanged_prefix_set) = self.take_changed_lower_subtries(&mut prefix_set);
 
+        // update metrics
+        self.update_hash_metrics(subtries.len());
```

Comment:
can we instead feature flag the call and the method, so that we don't have arguments prefixed with `_`?

Reply:
sure, that makes sense